code/modules/vending/toys.dm

```diff
@@ -25,8 +25,8 @@
 	armor = list("melee" = 100, "bullet" = 100, "laser" = 100, "energy" = 100, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
 	resistance_flags = FIRE_PROOF
 	refill_canister = /obj/item/vending_refill/donksoft
-	default_price = 150
-	extra_price = 300
+	default_price = 10
+	extra_price = 10
```

Comment:
Same as above, yet these ones can be obtained in a normal round with some efforts.

Reply:
Non-issue. The toylathe prints all of these, for free, with less effort to obtain than a stocked toy vendor.

Reply:
I'm going to nerf the toylathe to hell, it has been a nuisance anyway.